logging/exp/alias.go

```diff
@@ -0,0 +1,138 @@
+// Copyright 2023 The Authors (see AUTHORS file)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package exp
```

Comment:
The purpose of the aliasing is to save the mass import update in the future. But if this package is named `exp`, we'd probably get rid of it as well. Maybe having a separate package like `slogging`?

Reply:
I think I'd prefer to cut a release of `abcxyz/pkg`, then rename this to logging (deleting the other one).

Reply:
That works. I just thought `slog` => `slogging` :)

Reply:
Are we planning on a change to slog soon? Or not until it gradates from exp?

Reply:
I was planning on adding additional middleware on top of the pkg/ current and want to know if I should do so in zap or wait for slog

Reply:
We will move to slog when it lands in the stdlib (currently planned for the next Go release).

What kind of middleware? slog/zap are _mostly_ interoperable.